RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,325 @@
+    - [Making a Release](#making-a-release)
+        - [Releasing for Linux and Windows](#releasing-for-linux-and-windows)
+        - [Releasing for FreeBSD](#releasing-for-freebsd)
+        - [Releasing for Windows](#releasing-for-windows)
+        - [Releasing for MacOS](#releasing-for-macos)
+  - [Release issue](#release-issue)
+    - [Release Manager](#release-manager)
+    - [Release Issue Update](#release-issue-update)
+  - [Version Increment](#version-increment)
+     - [Version Increment Core](#version-increment-core)
+     - [Release Branch](#release-branch)
+     - [Plugins Version Increment](#plugins-version-increment)
+ - [Distribution Build](#distribution-build)
+ - [Release Candidate](#release-candidate)
+ - [Final Release](#final-release)
+    - [Pre-Release](#pre-release)
+    - [Release Notes](#release-notes)
+    - [Release](#release)
+    - [Post-Release](#post-release)
+
+## OpenSearch Release Process
+
+### Release Terminology and Knowledge Center
+
+#### Types of Manifests
+
+
+- Input Manifest: Uses as an initial input to drive the build workflow. These manifests are [located](https://github.com/opensearch-project/opensearch-build/tree/main/manifests) in build repo and gets created by [manifest workflow](https://github.com/opensearch-project/opensearch-build/tree/main/src/manifests_workflow).
+Once the manifest is created the maintainers of the build repo approves and merges it to `main` branch. The entire build repo runs on single branch `main`.
+
+- Build Manifest: Output of the [build workflow](https://github.com/opensearch-project/opensearch-build/tree/main/src/build_workflow), used as input to the [assemble workflow](https://github.com/opensearch-project/opensearch-build/tree/main/src/assemble_workflow). Sample [manifest](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/1.3.10/6126/linux/x64/tar/builds/opensearch-dashboards/manifest.yml).
+
+    ##### Sample Generated Build Manifests
+
+    | deb | rpm | tar | windows |
+    | --- | --- | --- | ------- |
+    | [os-arm64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/linux/arm64/deb/builds/opensearch/manifest.yml), [os-x64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/linux/x64/deb/builds/opensearch/manifest.yml)    |  [os-arm64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/linux/arm64/rpm/builds/opensearch/manifest.yml), [os-x64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/linux/x64/rpm/builds/opensearch/manifest.yml)   |  [os-x64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/linux/x64/tar/builds/opensearch/manifest.yml), [os-arm64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/linux/arm64/tar/builds/opensearch/manifest.yml)   |    [os-x64](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.11/7979/windows/x64/zip/builds/opensearch/manifest.yml)     |
```

Comment:
We should not use a unreleased version as example.
Also would prefer to use 2.x version links as those are more up to date.

Reply:
Updated peter, adding already released version links.